content/en/blog/2023/0000-00-00-kube-proxy-non-privileged.md

```diff
@@ -0,0 +1,113 @@
+---
+layout: blog
+title: "Kubernetes 1.29: Support for Running kube-proxy in a Non-privileged Container"
```

Comment:
We ran out of capacity to review and publish this as part of the post-release blog series.

How about:
```suggestion
title: "Kubernetes supports running kube-proxy in an unprivileged container"
```
?